pkg/crc/preflight/preflight_windows.go

```diff
@@ -40,7 +40,7 @@ var hypervPreflightChecks = []Check{
 		checkDescription: "Checking Windows edition",
 		check:            checkWindowsEdition,
 		fixDescription:   "Your Windows edition is not supported. Consider using Professional or Enterprise editions of Windows",
-		flags:            NoFix,
+		flags:            NoFix | StartUpOnly,
```

Comment:
Not sure the installer is checking for this?

Reply:
Thanks, its not checking for the windows edition currently, only the build number, will add an windows edition check in the msi launch conditions.

Reply:
check added in the msi in https://github.com/code-ready/crc/pull/2636/commits/7181db10c3408ec0b3706a03ace1c19e30353682